ingest/src/main/java/com/connexta/ingest/client/TransformClient.java

```diff
@@ -22,20 +23,22 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TransformClient.class);
```

Comment:
✏️ Should we start using the Lombok annotation for the loggers from now on?

Reply:
My preference is to use the annotation.